neo4j_utils/_driver.py

```diff
@@ -830,18 +830,17 @@ def db_status(
 
         name = name or self.current_db
 
-        query = f'SHOW DATABASES WHERE name = "{name}";'
+        query = 'SHOW DATABASES'
 
         with self.fallback():
 
-            try:
-                resp, summary = self.query(query=query)
-            except neo4j.exceptions.ClientError:
-                return None
-
-        if resp:
+            resp, summary = self.query(query=query)
+            databases = [record['name'] for record in resp]
 
+        if name in databases:
             return resp[0].get(field, resp[0])
```

Comment:
I'm not getting how this makes sense: nothing guarantees that the element at index `0` will be the database with `name`